### PR TITLE
Feat resource label

### PR DESCRIPTION
**Pull request by PabloAbara**
# Base PR
- PR based on main

# Context
- With this PR, we are implementing resource labels on learning unit resources endpoint. 

# Changelog
- Implement a method for Resource model that brings just required information of the Labels (id, name, description). 
- Update resource#index action on api controller to call resource_labels. 
- Add seeds to have several labels and resources with corresponding labels.
- Update rspec for resource index endpoint. 
- Update swagger   

# QA
- Launch Rails console inside docker with `docker compose exec web bash` after `docker compose up` on another terminal tab. 
- Make sure to either run new seeds or create a Label and ResourceLabel through rails console.
- If running seeds: `rails db:seed`. If this doesn't work you may have mislead prior migrations, so you could reset database by running: (1) `rails db:drop` `rails db:create` `rails db:migrate` `rails db:seed`
- Access to: http://localhost:3000/api/learning_units/1/resources to see the result of the new api call. 

You should see the resource_labels **array** to each resource that belongs to learning_unit with id = 1.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -91,6 +91,23 @@
 CompletedLearningUnit.create(user: ale, learning_unit: ruby)
 CompletedLearningUnit.create(user: ale, learning_unit: js)
 
+paid = Label.create(name: "Paid", description: "This resource is paid")
+media = Label.create(name: "Media", description: "Includes a video")
+article = Label.create(name: "Article", description: "Is an article")
+
+
+[github, ruby_udemy, react_redux].each do |resource|
+  ResourceLabel.create(resource_id: resource.id, label_id: paid.id)
```
Comment (jjsantos01) [inline db/seeds.rb:100]: Remember that you can write this also as `ResourceLabel.create(resource: resource, label: paid)`, which is a little bit concise.

**Review by jjsantos01 — APPROVED** [review]
Very clear PR. Tested on local and worked, although I had troubles with migrations. Followed the instructions and worked fine.

Just left a comment that you can attend if you want.

File: db/seeds.rb (hunk 1 above)
Comment (PabloAbara) [reply db/seeds.rb:100]: Yes! However, I tried to follow DRY and avoid repeat each line on any new resource label creation.

Comment (PabloAbara) [reply db/seeds.rb]: Done!

**Review by CRiquelme — APPROVED** [review]
Nice job!

**Pull request merged**